Makefile

```diff
@@ -43,10 +52,11 @@ images: venv build.stamp $(DRAWBOT_OUTPUT)
 
 clean:
 	rm -rf venv
-	find . -name "*.pyc" | xargs rm delete
+	find . -name "*.pyc" | xargs -r rm
```

Comment:
xargs rm delete fails in bash and zsh as per my testing. It throws this error:
[Error 'rm: missing operand' when using along with 'find' command](https://stackoverflow.com/questions/36617999/error-rm-missing-operand-when-using-along-with-find-command)

The GNU args compatible flag -r resolves the issue.
from man xargs
  -r, --no-run-if-empty
  If the standard input does not contain any nonblanks, do not run the command.  Normally, the command is run once   even if there is no input.  This option is a GNU extension